src/batUnix.mlv

```diff
@@ -30,6 +30,27 @@ include Unix
 ##V<4.2##let send_substring = send
 ##V<4.2##let sendto_substring = sendto
 
+##V<4.3##let sleepf (timeout: float): unit =
+##V<4.3##  let _, _, _ = select [] [] [] timeout in
```

Comment:
You probably want to restart_on_EINTR.

Also, have you considered `Thread.delay` ?

Reply:
I'll have a look at that. Thanks for the review.

Reply:
Concerning thread.delay, it is implemented as Unix.sleepf in recent OCaml versions.

Reply:
the implementation became a little bit more complex

Reply:
I'm assuming `Thread.delay` does not use `Unix.sleepf` when V<4.3, so wouldn't that be easier and more reliable to just call `Thread.delay` then? Unless we do not want to link the Unix to the Thread module, maybe?

Reply:
They use thread_delay from scheduler.c, which calls schedule_thread, which looks like a real mess.
But, is the semantics of delaying a thread the same as making a process sleep?
I think a process could have several threads, in theory.